.github/workflows/run-expected-count-script-on-pr.yml

```diff
@@ -0,0 +1,34 @@
+name: Run check expected count script on changed queries
+
+on:
+  pull_request:
+    paths:
+      - '**/*.rq' # Only run when .rq files are modified
+
+jobs:
+  check_files:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v3
+
+    - name: Set up Python
+      uses: actions/setup-python@v4
+      with:
+        python-version: '3.x'
+
+    - name: List changed .rq files
+      id: list-files
+      run: |
+        # Get a list of .rq files changed in the PR
+        CHANGED_FILES=$(git diff --name-only ${{ github.event.before }} ${{ github.sha }} -- '*.rq')
+        echo "files=$CHANGED_FILES" >> $GITHUB_ENV
+
+    - name: Run script on each .rq file
```

Comment:
I would expect this to always pass, without a check looking for errors in the output. I would suggest `grep` or `find` to lookup the stats at the bottom of the table.

Reply:
I see, the process do currently not fail for non-runtime errors.

Reply:
I think that's fine for now, also because we're not on a baseline where all data is good. For example, my improvement of the Austrian query in #451 would still fail. Once we get to that baseline again, adding such functionality to the check-script would be a good next step. Until then, one could still see in the workflow how it went.

Reply:
All good points!